lib/phoenix/code_reloader.ex

```diff
@@ -49,26 +51,86 @@ defmodule Phoenix.CodeReloader do
   API used by Plug to invoke the code reloader on every request.
   """
   def call(conn, opts) do
-    case opts[:reloader].(conn.private.phoenix_endpoint) do
-      :ok ->
+    {conn, res, output} = call_reloader(conn, opts[:reloader])
+
+    case {res, feedback_started?(conn)} do
+      {:ok, true} ->
+        Plug.Conn.chunk(conn, "Done!<br />\n")
+        Plug.Conn.chunk(conn, ~s(<meta http-equiv="refresh" content="0">\n))
```

Comment:
I am worried about the impact of this line on developer using Windows. Mix tracks mtimes and they are very fragile on Windows. Some users report recompilation of some files on every access and this refresh may put those applications in an infinite loop.